third_party/terraform/website/docs/r/cloudfunctions_function.html.markdown

```diff
@@ -13,6 +13,12 @@ Creates a new Cloud Function. For more information see
 and
 [API](https://cloud.google.com/functions/docs/apis).
 
+~> **Warning:** As of November 1, 2019, newly created Functions are
+private-by-default and will require [appropriate IAM permissions](https://cloud.google.com/functions/docs/reference/iam/roles)
+to be invoked. See below examples for how to set up the appropriate permissions,
+or view the Terraform [IAM resources](/docs/r/cloudfunctions_cloud_function_iam.html)
```

Comment:
nit: Terraform is redundant &  link text is a bit more clear as "IAM resources for Cloud Functions" or "Cloud Functions IAM resources".

Reply:
:+1: